docs/embeddings/configuration/cloud.md

```diff
@@ -26,21 +26,30 @@ Container/bucket/directory/repository name.
 
 ## Cloud object storage configuration
 
-In addition to the above common configuration, the cloud object storage provider has the following additional configuration parameters.
+In addition to the above common configuration, the cloud object storage provider has the following additional configuration parameters. The number of these configs which are need depend upon your cloud provider. Some cloud providers do not need any of these configurations and can use implicit authorization with service accounts.
```

Comment:
Revise to:

In addition to the above common configuration, the cloud object storage provider has the following additional configuration parameters. Note that some cloud providers do not need any of these parameters and can use implicit authorization with service accounts.